packages/components/README.md

```diff
@@ -147,13 +147,13 @@ We use [typedoc](https://www.npmjs.com/package/typedoc) for generating our docum
 All exported components, methods, interfaces, etc. should include
 documentation.  You can use the supported [JavaDoc tags](https://typedoc.org/guides/doccomments/) to provide additional explanations for parameters, return values, etc. as well as for indicating that some objects should not have documentation generated for them.
 
-You can generate the docs locally by running the following command in the ```labkey-ui-components``` directory.
+You can generate the docs locally by running the following command in the ```labkey-ui-components/packages/components``` directory.
 * ``yarn run build:docs``
 
 **N.B.** This command produces a good number of errors from Typescript not being able to resolve classes and such.
 Try to eliminate as many of these as possible before deploying the docs.
 
-To publish the documentation to [GitHub pages](https://labkey.github.io/labkey-ui-components/) use the following command in the ```labkey-ui-components``` directory:
+To publish the documentation to [GitHub pages](https://labkey.github.io/labkey-ui-components/) use the following command in the ```labkey-ui-components/packages/components``` directory:
```

Comment:
When the time comes, we'll likely want to surface the link to the docs in a more prominent place.

Reply:
Agreed.